client/components/CandidateTestPlanRun/queries.js

```diff
@@ -0,0 +1,152 @@
+import { gql } from '@apollo/client';
+
+export const ADD_VIEWER_MUTATION = gql`
+    mutation AddViewerMutation($testPlanVersionId: ID!, $testId: ID!) {
+        addViewer(testPlanVersionId: $testPlanVersionId, testId: $testId) {
+            username
+        }
+    }
+`;
+
+export const PROMOTE_VENDOR_REVIEW_STATUS_REPORT_MUTATION = gql`
+    mutation UpdateVendorReviewStatusReport(
+        $testReportId: ID!
+        $reviewStatus: String!
+    ) {
+        testPlanReport(id: $testReportId) {
+            promoteVendorReviewStatus(vendorReviewStatus: $reviewStatus) {
+                testPlanReport {
+                    id
+                    vendorReviewStatus
+                }
+            }
+        }
+    }
+`;
+
+export const CANDIDATE_REPORTS_QUERY = gql`
+    query CandidateReportsQuery($testPlanVersionId: ID!, $atId: ID!) {
+        me {
+            id
+            roles
+            username
+        }
+        testPlanReports(
+            statuses: [FINALIZED]
+            testPlanVersionId: $testPlanVersionId
+            atId: $atId
+        ) {
+            id
+            candidateStatusReachedAt
+            recommendedStatusTargetDate
+            vendorReviewStatus
+            issues {
+                author
```

Comment:
The mockup also includes a hyperlink on the text which says something like `x people have left feedback for this test`. My guess is that should be to the issue(s) (although I'm not sure how to account for multiple links visually, a pop up or so of applicable links? But this may be something to discuss internally).

But note that `link` is available in the query to get access to those GitHub issue links

Reply:
@s3ththompson @isaacdurazo how should we account for the multiple links in the case of `x people have left feedback for this test`?

Reply:
It is possible. We'd just have to map a link with the right query to the component.